src/index.js

```diff
@@ -1,12 +1,14 @@
 import 'babel-polyfill';
 import 'source-map-support/register';
 
-import debug from 'debug';
+import debugLog from 'debug';
 import fs from 'graceful-fs';
 import mkdirp from 'mkdirp';
```

Comment:
Is this really a run-time dependency for fs-wrapper? We may want to consider a couple of options here - one is to remove this dependency all-together (i.e. take out the methods in the module that utilize it or write our own version of it in this module). If that does not seem tenable, then this dependency needs to be added to the `dependencies` list in the `package.json` file.